modifications to the content (#1)
who-owns-lausanne/who-owns-lausanne.github.io: index.md

time-consuming and tedious task. If you have just arrived in Lausanne, you don't
know where to look. Is it cheaper on the hill or near the university? You don't
know all the parameters that come into play and while running from one flat
viewing to another you start to wonder: who owns all this real estate you would
like to live in?

#### Goal

We want to bring some light into the opaque world of real estate. By leveraging
openly accessible data our goal is to understand **how rent prices are
influenced by the type of the owner, the geographical situation, or other
factors**. This article should serve as a head start for people searching
affordable rents in Lausanne.

#### Key insights

In particular you will be enlightened by the following real estate insights
and you will understand how these were obtained. Even if Lausanne is not
New York we needed to think of it in smaller units. Therefore, our results
divide the city into its neighbourhoods – in french
[_quartiers_][quartiers_lausanne].

> - the _quartiers_  _Montriond_ and _Ouchy_  are clearly more expensive

Reviewer: ```suggestion
- The _quartiers_  _Montriond_ and _Ouchy_  are clearly more expensive
```
all others start with capital letters

  than the rest.

- The city center is mostly owned by corporations,
  _Chailly_ is mostly owned by privates.

> - There is no direct relation between owner type and prices. The market seems to

Reviewer: ```suggestion
- There is no direct relation between the owner type and prices. The market seems to
```

Author: je pense que ma version est aussi correcte

Reviewer: Sure, c'était juste le flow. Mais ça marche aussi.

  adjust prices uniformly over the ownership types.

> - The price per square meter of rent is strongly influenced by the distance to the

Reviewer: ```suggestion
- The price per square meter of a flat is strongly influenced by the distance to the
```

 - cooperatives: _registered cooperative companies like Migros, la Mobilière etc._

 - foundations and associations: _for example the olympic foundation for
cultural heritage._

 - PPE: _the single flats in a building are owned by different people – in french

>    proriété par étage_.

Reviewer: ```suggestion
_   proriété par étage.
```
French words are italic, but in italic text, the new italic is again normal text...

Author: ok but weird

>  - individual privates: _The whole building is owned by a private citizen._

Reviewer: ```suggestion
 - individual privates: _the whole building is owned by a private citizen._
```

Here all others start with downcase.

statistically correct answer, we evaluated a simple linear model with
[linear regression]. The model tries to find significantly different mean rent
prices depending on the ownership type of a parcel.
This model howver does not give a meaningful result. It seems like our
hypothesis that rents are different if the property is for example owned by a
corporation has no factual basis, and that the small differences in the mean
rents of each ownership type just arise from random noise. After all,
we only have a small number of offers compared to the number of parcels.

#### Geographical factors

But if the owner doesn't influence the price what does? Why are prices
different? You can get an intuition for this by looking at the smoothed rent
map. Clearly, the parts near the lake are more red. Our next linear regression
therefore analyses the dependence of the price on the distance to the port of
Lausanne, [_Ouchy_]. As you can observe in the plot to the left, there is a
visible correlation. Our model can confirm with a high significance that the

> more you go toward the country-side, the cheaper flats get.

Reviewer: ```suggestion
more you go towards the country-side, the cheaper flats get.
```
